pickhardtpayments/ChannelGraph.py

```diff
@@ -46,3 +53,66 @@ def get_channel(self, src: str, dest: str, short_channel_id: str):
         if self.network.has_edge(src, dest):
             if short_channel_id in self.network[src][dest]:
                 return self.network[src][dest][short_channel_id]["channel"]
+
+
+def lnd2cln_json(channel_graph_json):
+    """
+    Converts the channel graph json from the LND format to the c-lightning format
+    """
+
+    LND_CLN_POLICY_MAP = {
+        "time_lock_delta": "delay",
+        "min_htlc": "htlc_minimum_msat",
+        "fee_base_msat": "base_fee_millisatoshi",
+        "fee_rate_milli_msat": "fee_per_millionth",
+        "disabled": "active",
+        "max_htlc_msat": "htlc_maximum_msat",
+        "last_update": "last_update",
+    }
+
+    def _add_direction(src, dest, lnd_channel, cln_channel):
+        cln_channel["source"] = lnd_channel[src + "_pub"]
```

Comment:
in `Channel.py` I have a class called `ChannelFields` that defines constants for those keys. Maybe we can use it to access fields and keys in  `cln_channel` and potentially do something similar for `lnd`?  but maybe that is also a bit overengineered?

Reply:
Thanks! Replaced the hard-coded fields with ChannelFields in a174641
I'm not sure if we should define something similar to ChannelFields for other formats, I think it might be nicer to convert other formats to the cln format (ChannelFields) as it would prevent a lot of overlap which can happen if we define a ChannelFields for each format separately. Could you please share your thoughts about it?

Reply:
I think that is reasonable. for now. The fields of other implementations are only being used here. I think on the long term if we also had class called `LNDChannelFields` we could have an automated test to see if lnd changed their API against our class but I think that is overkill. So I think the current fix is ok